spacy/tokens/token.pyx

```diff
@@ -237,11 +237,6 @@ cdef class Token:
         index into tables, e.g. for word vectors."""
         return self.c.lex.id
 
-    @property
-    def string(self):
```

Comment:
I'm pleased to finally be rid of this. It's an example of something that I _do_ feel really gets in the way.